pkg/minikube/cluster/cluster.go

```diff
@@ -413,19 +414,23 @@ func GetHostDockerEnv(api libmachine.API) (map[string]string, error) {
 
 // GetHostLogs gets the openshift logs of the host VM.
 // If follow is specified, it will tail the logs
-func GetHostLogs(api libmachine.API, follow bool) (string, error) {
+func GetHostLogs(api libmachine.API, follow bool, tail int64) (string, error) {
 	host, err := CheckIfApiExistsAndLoad(api)
 	if err != nil {
 		return "", err
 	}
 
-	if follow {
+	if follow || tail != -1 {
 		c, err := host.CreateSSHClient()
 		if err != nil {
 			return "", err
 		}
 
-		err = c.Shell(logsCmdFollow)
+		if follow {
```

Comment:
what about tail and follow ?

Reply:
@chmouel Never though :thinking: , will check :)

Reply:
@chmouel Unfortunately, not working. We get only one feature at a time. Thanks.

Reply:
@budhram Really? I have it working here, anyway feel free to send this over i'll send a PR,

Reply:
@chmouel can you please send the PR and we will close this one then if it is working as expected.